docs/INTEGRATIONS.rst

```diff
@@ -0,0 +1,642 @@
+Integrations
```

Comment:
"Installing Duplicity and its dependencies" and associated "Testing your installation" and "Automating Backps" are not nested under "Integration", it seems